content/en/security_platform/cspm/getting_started.md

```diff
@@ -19,30 +19,200 @@ Cloud Security Posture Management is not currently available in this site.
+1. Navigate to **Security** > **Setup and Configuration**.
+2. Follow the [in-app instructions][3] to activate CSPM for your account.
+3. On the **Setup and Configuration** > **Cloud Providers** tab, click the **AWS** tile.
+4. To enable CSPM for an AWS account, turn on the **Collect Resources** toggle.
 
-## Further reading
+#### AWS integration tile
 
-{{< partial name="whats-next/whats-next.html" >}}
+1. On the AWS integration tile, select an AWS account and click **Resource Collection**.
+2. Select **Cloud Security Posture Management Collection** to enable resource collection for CSPM.
+3. Click **Save**.
 
-[1]: https://app.datadoghq.com/security/configuration?config_k9_configuration=true&detect-threats=apache&secure-cloud-environment=amazon-web-services&secure-hosts-and-containers=kubernetes&selected-products=compliance_monitoring
-[2]: /security_platform/default_rules#cat-posture-management-cloud
-[3]: /security_platform/default_rules/#cat-posture-management-infra
+**Note**: To disable CSPM, turn off the **Collect Resources** toggle for your AWS accounts. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://docs.datadoghq.com/integrations/amazon_web_services/
+[2]: /integrations/amazon_web_services/?tab=roledelegation#cloud-security-posture-management
+[3]: https://app.datadoghq.com/security/configuration
+
+{{% /tab %}}
+
+{{% tab "Azure" %}}
+
+### Set up the Datadog Azure integration
+
+If you haven't already, set up the [Microsoft Azure integration][1].
+
+### Enable CSPM for Azure
+
+Use one of the following methods to enable CSPM for your Azure subscriptions:
+
+#### Security Setup & Configuration
+
+1. Navigate to **Security** > **Setup and Configuration**.
+2. Follow the [in-app instructions][2] to activate CSPM for your account.
+3. On the **Setup and Configuration** > **Cloud Providers** tab, click the **Azure** tile.
+4. Enable CSPM for your Azure subscriptions by turning on the **CSPM Enabled** toggle.
+
+#### Azure integration tile
+
+1. On the Azure integration tile, select an Azure app.
+2. Under **Resource Collection**, select the **Enable resource collection for Cloud Security Posture Management** checkbox.
+3. Click **Update Configuration**.
+
+**Note**: To disable CSPM, turn off the **CSPM Enabled** toggle for your Azure subscriptions. Your previous findings and homepage will still be available in-app, but you will not incur additional billing costs.
+
+[1]: https://docs.datadoghq.com/integrations/azure
+[2]: https://app.datadoghq.com/security/configuration
+
+{{% /tab %}}
+
+{{% tab "GCP" %}}
+
+### Set up the Datadog GCP integration
+
+If you haven't already, set up the [Google Cloud Platform integration][1] and make sure that you have successfully completed the steps for enabling [metric collection][2].
```

Comment:
we've found that users often forget this step or do it partially, so we may want to highlight this even more visually speaking?

Reply:
Which part? The metric collection?

Reply:
enabling all relevant GCP APIs